RELEASE.md

```diff
@@ -7,7 +7,12 @@
 * `tf.lite`:
   * Remove `experimental.nn.dynamic_rnn`, `experimental.nn.TfLiteRNNCell` and  `experimental.nn.TfLiteLSTMCell` since they're no longersupported. It's  recommended to just use [keras lstm](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTM) instead.
 
-* Keras been split into a separate PIP package (`keras`), and its code has been moved to the GitHub repository[keras-team/keras](http://github.com/keras-team/keras). The API endpoints for `tf.keras` stay unchanged, but are now backed by the `keras` PIP package. The existing code in tensorflow/python/keras is a staled copy and will be removed in future release (2.7). Please remove any imports to `tensorflow.python.keras` and replace them with public tf.keras API instead.
+* `tf.keras`:
+  * Keras been split into a separate PIP package (`keras`), and its code has been moved to the GitHub repository[keras-team/keras](http://github.com/keras-team/keras). The API endpoints for `tf.keras` stay unchanged, but are now backed by the `keras` PIP package. The existing code in tensorflow/python/keras is a staled copy and will be removed in future release (2.7). Please remove any imports to `tensorflow.python.keras` and replace them with public tf.keras API instead.
+  * The methods `Model.to_yaml()` and `keras.models.model_from_yaml` have been
+    replaced to raise a `RuntimeError` as they can be abused to cause arbitrary
+    code execution. It is recommended to use JSON serialization instead of YAML,
+    or, a better alternative, serialize to H5.
```

Comment:
```suggestion
  * The methods `Model.to_yaml()` and `keras.models.model_from_yaml` have been replaced to raise a `RuntimeError` as they can be abused to cause arbitrary code execution. It is recommended to use JSON serialization instead of YAML, or, a better alternative, serialize to H5.
```